tests/test-images/system-container-files/config.json.template

```diff
@@ -114,11 +114,9 @@
 	],
 	"hooks": {},
 	"linux": {
-		"capabilities": [
-			"CAP_AUDIT_WRITE",
-			"CAP_KILL",
-			"CAP_NET_BIND_SERVICE"
-		],
+	         "capabilities": {
+			"bounding": [${ALL_PROCESS_CAPABILITIES}]
```

Comment:
Is ``bounding`` supported for all the current versions of runc we are supporting?

Reply:
Does this mean we will be allowing all capabilities supported by a system from now on?

Reply:
we already dropped the support for the older OCI spec file support where there was no differentiation among the different kind of capabilities (i.e. `capabilities` was a list not a dictionary). 
runc honors this setting.

We don't allow all capabilities, this is meant only for privileged containers like CRI-O/Docker that anyway are already listing all the capabilities in their config.json.template file so that we don't need to carry two different lists on Fedora and RHEL since the capabilities sets are different.

/cc @rhatdan